Makefile

```diff
@@ -706,4 +707,8 @@ sbom: sbom/assets/kurl-sbom.tgz
 
 .PHONY: tag-and-release
 tag-and-release: ## Create tags and release
+ifneq "$(COMMIT_ID)" ""
+	@./bin/tag-and-release.sh --commit-id=$(COMMIT_ID)
```

Comment:
Do you need to pass --outdated=1?

Reply:
It's worth enforcing to have a clean git tree prior to pushing a tag so I will _not_ override the default behavior.